app/frontend/Calendar/EditEvent/RepeatBox.svelte

```diff
@@ -186,35 +186,6 @@
     }
     return new RecurrenceRule(init);
   }
-
-  // TODO Call from save()
-  export function confirmAndChangeRule(): boolean {
```

Comment:
Please try to keep the UI functions regarding recurrence in RepeatBox.svelte, as far as possible.

Reply:
I see. Too bad.